pkg/process/process.go

```diff
@@ -286,8 +286,8 @@ func AddCloneEvent(event *tetragonAPI.MsgCloneEvent) error {
 		if pi.process.Pod != nil && pi.process.Pod.Container != nil {
 			pi.process.Pod.Container.Pid = &wrapperspb.UInt32Value{Value: event.NSPID}
 		}
+		procCache.Add(pi)
```

Comment:
Can you add a short commit message (or even comment) about why is this change needed?
Is `AddCloneEvent` called multiple times until `pi.process` is not nil?

Reply:
done!